### ISSUE TITLE
[DOCUMENTATION] Create guide to contribute to Dev Protocol 

### ISSUE THREAD

**Issue by vinzvinci** [dev-protocol/community#147]
### What would you like to share?

Only create using markdown, just telling the guide on how to start contributing to Dev Protocol

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MatteuSan** [comment]
Yes please! This can also serve as a base template for specific repositories. (And selfishly, it helps me for Hashi lmao. I don't have any idea how to write good contributing guidelines)

**Comment by vinzvinci** [comment]
Thanks, added this to trello.
